### PR TITLE
JSON policy sets

### PR DESCRIPTION
**Pull request by khieta**
*Issue #, if available:*

*Description of changes:*

Add description of the new JSON format for policy formats, which will be released as part of 4.0.

(The current plan is to merge 4.0-related changes to the `staging-4.0` branch, and then pull these back to `main` once 4.0 is released.)

### CODE REVIEW

File: docs/collections/_policies/json-format.md [hunk 1]
```diff
@@ -1116,3 +1123,102 @@ JSON representation
...
+
+```cedar
+permit (
+    principal == User::"12UA45",
+    action == Action::"view",
+    resource in Folder::"abc"
+);
+
+permit (
+    principal == User::"12UA45",
```
Comment (mwhicks1) [inline docs/collections/_policies/json-format.md:1141]: Nit -- change this one to a `forbid` just to make the example a little more expansive?

**Review by mwhicks1 — APPROVED** [review]

File: docs/collections/_policies/json-format.md (hunk 1 above)
Comment (khieta) [reply docs/collections/_policies/json-format.md]: Done

File: docs/collections/_other/doc-history.md [hunk 2]
```diff
@@ -12,11 +12,12 @@ The following table describes major documentation updates for Cedar.
 
 | Description | Date |
 | --- | --- |
+| Added [JSON policy set format](../policies/json-format.html#policy-set-format) sub-topic | TBD |
 | Added [Human-readable schema format](../schema/human-readable-schema.html) topic | February 21, 2024 |
-| Added [`is` operator](../policies/syntax-operators.html) | December 15, 2023 |
+| Added [`is` operator](../policies/syntax-operators.html#operator-is) sub-topic | December 15, 2023 |
```
Comment (john-h-kastner-aws) [inline docs/collections/_other/doc-history.md:17]: Could go directly to `main`

File: docs/collections/_policies/json-format.md [hunk 3]
```diff
@@ -1,20 +1,27 @@
 ---
 layout: default
-title: Programmatic policy creation using JSON
+title: JSON policy format
```
Comment (john-h-kastner-aws) [inline docs/collections/_policies/json-format.md:3]: Better title definitely. (Could also go to `main`)

**Review by john-h-kastner-aws — APPROVED** [review]
Looks good. A couple changes here could also go to `main` since `4.0` is a little ways off

**Pull request merged**